docs/get-started/mapbox-tokens.md

```diff
@@ -1,12 +1,12 @@
 # About Mapbox Tokens
 
-`react-map-gl` itself is open source and free. It provides a React wrapper for `mapbox-gl` or derived projects.
-Depending on which Mapbox GL JS version (or fork) you use, you may need a Mapbox token. You will need a Mapbox token if you use:
+`react-map-gl` itself is open source and free. It provides a React wrapper for `mapbox-gl`, `maplibre-gl` or compatible projects.
+Depending on which base map library you use, you may need a Mapbox token. You will need a Mapbox token if you use:
 
 -  [mapbox-gl@>=2.0.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v2.0.0) and above - requires a mapbox access token in order to access the map renderer, and generates billable events regardlesss of whether you are displaying your own maps.
--  `mapbox-gl@1.x` or [maplibre-gl](https://github.com/maplibre/maplibre-gl-js) - requires an access token only if you load the map styles and tiles from Mapbox's data service. See "Display Maps Without A Mapbox Token" section below for using non-Mapbox tiles.
+-  `mapbox-gl@1.x` or `maplibre-gl` - requires an access token only if you load the map styles and tiles from Mapbox's data service. See "Display Maps Without A Mapbox Token" section below for using non-Mapbox tiles.
```

Comment:
MapLibre 2.x removed the mapbox token. I seem to recall a plugin was made to support mapbox token in new versions.